package.json

```diff
@@ -21,9 +21,10 @@
     "test:integration:microblocks": "npm run devenv:deploy:pg -- -d && cross-env NODE_ENV=development jest --config ./jest.config.microblocks.js --coverage --no-cache --runInBand; npm run devenv:stop:pg",
     "test:integration:tokens": "npm run devenv:deploy -- -d && cross-env NODE_ENV=development jest --config ./jest.config.tokens.js --coverage --no-cache --runInBand; npm run devenv:stop",
     "git-info": "echo \"$(git rev-parse --abbrev-ref HEAD)\n$(git log -1 --pretty=format:%h)\n$(git describe --tags --abbrev=0)\" > ./.git-info",
-    "build": "npm run git-info && rimraf ./lib && tsc -p tsconfig.build.json",
+    "build": "npm run git-info && npm run build:api-documentation && rimraf ./lib && tsc -p tsconfig.build.json",
```

Comment:
I'm conflicted about making this a build requirement for the root project. For example, one of the doc generators requires the java runtime. Seems like the dockerfiles (and whatever else) should just explicitly call a build:docs command after this initial one.

Reply:
But it is important that this command runs before the server starts. So we have the HTML ready to be rendered. Through this, upon running `dev:integrated`, it will run `build` which runs `build:api-documentation`. 

For explicitly running it, do you think I should print an error message like `Please run the command: npm run build:docs and restart your server` for endpoint `/doc`, in case the server starts without building documents (i.e. HTML file was not created)?

Reply:
The other projects like Clarinet do not compile the API from source. The docker images are used, so those docker images should be responsible for ensuring the docs are built and included. For those working on this repo, it doesn't make sense to require that the core `build` script do this extra work. 

> For explicitly running it, do you think I should print an error message like ...

Yes if the doc endpoint is supposed to be served, but the `index.html` file doesn't exist, this would be a helpful error response.